packages/account/src/Sections/Verification/Helpers/verification.js

```diff
@@ -1,19 +1,26 @@
-export const populateVerificationStatus = account_status => {
-    const { attempts, document, identity, needs_verification } = account_status.authentication;
+const populateVerificationStatus = account_status => {
+    const { attempts, document, identity, needs_verification } = {
+        ...account_status?.authentication,
+        attempts: account_status?.authentication?.attempts || {},
+        needs_verification: account_status?.authentication?.needs_verification || {},
+    };
```

Comment:
I assume this is modified to set default values to `attempts` and `needs_verification` in case those props are `undefined`.

There is a better way to set default values, we can do it during destructing.
```
const { attempts = {}, document, identity, needs_verification = {} } = account_status.authentication;
```

Reply:
Thanks @balakrishna-binary for your suggestion,
I have to spread it to an object because in some cases for example in the initial render we call this function but account_status is undefined. I tested your suggestion on code and it throws error like the picture that I attached
 
![Screenshot from 2021-11-09 07-36-34](https://user-images.githubusercontent.com/81898967/140861638-0f9fba51-ef92-4029-8e37-0a4667c8b425.png)
The purpose of `account_status.authentication` spread  in object and also default empty object is to have a workaround about undefined account_status that it's possible to get from different places.
Another approach would be to write an if condition whenever we are using populateVerificationStatus   that is not a good idea IMO.

Reply:
@ali-hosseini-fs Though this is one way to circumvent undefined `account_status` object, but the main reason this occurs is because the `account_status` data is not fully fetched yet. One way is to wrap the `populateVerificationStatus()` in a `WS.authorized.getAccountStatus()` callback to be sure the `account_status` is fully populated before destructuring. In this case you can omit the fallbacks and fix the problem at its roots.